src/Cabinet/Cabinet+NOTIFICATION.cs

```diff
@@ -23,17 +23,17 @@ public unsafe struct NOTIFICATION
             /// <summary>
             /// A null-terminated string.
             /// </summary>
-            public char* psz1;
+            public sbyte* psz1;
```

Comment:
These `pszN` values represent an ANSI-encoded string, not binary data. I used `sbyte` so that you can pass this value directly to the `string(sbyte*)` constructor.

Reply:
TIL. Why in the world would .NET have decided to use `sbyte` in this API?
I would still prefer to use `byte*` anyway as that's what people everywhere else use for data. Instead of using `new string`, you can decode the buffer using the [`Encoding.GetString`](https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.getstring?view=netcore-3.1#System_Text_Encoding_GetString_System_Byte__System_Int32_) method.

Reply:
So, I attempted to change it to `byte*` instead, _but_ this struct doesn't expose the length of the strings; you need to rely on them being null-terminated.
There are `Encoding.GetString(byte*,int)` overloads which require you to pass the string length, but I couldn't find any overload which parses a null-terminated string. So I'd suggest to leave it at `sbyte*` for now. I can still change it if you prefer, though.

Reply:
Ok. I don't have a better option and I get that you want these strings to be convenient to decode, so we'll go with what you have.